trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -87,6 +94,7 @@ describe('controllers - case', () => {
           tfm: mockDeal.tfm,
           activePrimaryNavigation: 'manage work',
           activeSubNavigation: 'deal',
+          successMessage: mockSuccessBannerMessage,
```

Comment:
Could this be simpler perhaps? i.e:

```ts
successMessage: getFlashSuccessMessage(mockReq),
```

Reply:
I've made it a tad simpler, have a look and see what you think :) I quite like keeping it as checking the expected string gets passed in, as opposed to calling a function - I think it's easier to read the test as and see what object we're expecting without having to work out what `getFlashSuccessMessage` does.